android/jni/IndoorAtlas/IndoorAtlasLocationManager.cpp

```diff
@@ -0,0 +1,118 @@
+//// Copyright eeGeo Ltd (2012-2015), All Rights Reserved
+
+#include "IndoorAtlasLocationManager.h"
+#include "AndroidAppThreadAssertionMacros.h"
+#include "LatLongAltitude.h"
+
+namespace ExampleApp
+{
+    namespace IndoorAtlas
+    {
+    	IndoorAtlasLocationManager::IndoorAtlasLocationManager(ExampleApp::IndoorAtlas::IndoorAtlasLocationService* indoorAtlasLocationService
```

Comment:
This type has really poor formatting, it's whole PR has poor formatting, but this is probably the worst.

Please setup your IDE's tabs-as-spaces indentation level to match the rest of EEA and run a format spank on this comment.